test/powershell/Modules/Microsoft.PowerShell.Utility/Sort-Object.Tests.ps1

```diff
@@ -230,8 +230,9 @@ Describe 'Sort-Object Top and Bottom Unit Tests' -Tags 'CI' {
 	{
 		param($nSortEntry, $fullSortEntry)
 		if ($nSortEntry -is [System.Array]) {
-			# Arrays must be compared differently than everything else
-			$nSortEntry.Equals($fullSortEntry) | Should Be $true
+			# Arrays are compared using reference equality to ensure that the original array was
+			# moved to the correct position in both sorts; value equality doesn't verify this
```

Comment:
My first thought was about Compare-Object, because many tests using it, but Sort-Object is specific. So maybe for the future _explicitly_ add: 'Compare-Object is not applicable here!'